app/views/account/your-details/subjects.html

```diff
@@ -38,8 +38,7 @@
     </div>
     <div class="govuk-grid-column-two-thirds-from-desktop">
       <h1 class="govuk-heading-l">{{ subHeading }}</h1>
-        <p>Here is the status of all the subjects and occupational areas you applied for. </p>
-        <p>You have been partially approved for one subject, and unsuccessful for 1 subject.</p>
+        <p>Here are the outcomes for all the subjects and occupational areas, qualification levels and activities you applied for. </p>
```

Comment:
I'd change this to "Outcomes for your subjects and occupational areas, qualifications, levels and activities you have applied.